frontend/src/scenes/events/EventsVolumeTable.tsx

```diff
@@ -158,6 +187,53 @@ export function VolumeTable({
     )
 }
 
+export function VolumeTableRecordDescription({
+    record,
+}: {
+    record: EventDefinition | PropertyDefinition
+}): JSX.Element {
+    const [newDescription, setNewDescription] = useState(record.description)
+    const [bordered, setBordered] = useState(false)
+    const { updateEventDefinition } = useActions(eventDefinitionsLogic)
+
+    return (
+        <div style={{ display: 'flex', minWidth: 300, marginRight: 32 }}>
+            <Input.TextArea
+                className="definition-description"
+                placeholder="Click to add description"
+                onClick={() => setBordered(true)}
+                bordered={bordered}
+                maxLength={400}
+                style={{ padding: 0, marginRight: 16, minWidth: 300 }}
+                autoSize={true}
+                value={newDescription || undefined}
+                onChange={(e) => setNewDescription(e.target.value)}
+            />
+            {bordered && (
+                <>
+                    <Button
+                        style={{ marginRight: 8 }}
+                        size="small"
+                        type="primary"
+                        onClick={() => updateEventDefinition(record.id, newDescription)}
```

Comment:
Per typing above you're accepting a `PropertyDefinition` too but you're posting to the events endpoint. Probably at this point worth only supporting events?

Reply:
yeah, I'll add the property stuff in a follow up PR!